src/Symfony/Component/Mailer/Bridge/Google/Transport/GmailSmtpTransport.php

```diff
@@ -20,9 +20,9 @@
  */
 class GmailSmtpTransport extends EsmtpTransport
 {
-    public function __construct(string $username, string $password, EventDispatcherInterface $dispatcher = null, LoggerInterface $logger = null)
+    public function __construct(string $username, string $password, int $port, EventDispatcherInterface $dispatcher = null, LoggerInterface $logger = null)
```

Comment:
i do not know if you can change this like this according to BC, moreover on sf v5.4

Reply:
Hm ok understand. But in which branch i can develop this? 6.1?